ChatCore/Models/MessagesRequest.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MessagesRequest.swift
+//  STRVChatCore
+//
+//  Created by Daniel Pecher on 23.02.2021.
+//
+
+import Foundation
+
+public struct MessagesRequest {
```

Comment:
I can't build the example project because this file is not part of `ChatCore` framework